src/base/utils.ml

```diff
@@ -23,18 +78,6 @@ struct
 
   open Runners.Make (Backend) (Checked) (As_prover) (Runner)
 
-  module Typ = struct
-    include Types.Typ.T
-    module T = Typ.Make (Checked_intf.Unextend (Checked))
-    include T.T
-
-    type ('var, 'value) t = ('var, 'value, Field.t, Cvar.t) T.t
-
-    let unit : (unit, unit) t = unit ()
-
-    let field : (Cvar.t, Field.t) t = field ()
-  end
-
```

Comment:
remove reconstructing `Typ` when it's already constructed in snark0 (exact copy/paste here)